src/supa/connection/provider/server.py

```diff
@@ -167,24 +179,50 @@ def ReserveCommit(
         Returns:
             A response telling the caller we have received its commit request.
         """
-        log = logger.bind(method="ReserveCommit")
-        log.info("Received message.", request_message=pb_reserve_commit_request)
+        connection_id = UUID(pb_reserve_commit_request.connection_id)
+        log = logger.bind(method="ReserveCommit", connection_id=str(connection_id))
+        log.debug("Received message.", request_message=pb_reserve_commit_request)
 
-        from supa import scheduler
+        from supa.db.session import db_session
```

Comment:
top level imports

Reply:
Some components cannot be be imported globally because they depend on the global settings that have not yet been initialised properly.